save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/demo/CpgView.kt

```diff
@@ -176,7 +175,7 @@ val cpgView: VFC = VFC {
             val alertStyle = if (cpgResult.logs.isNotEmpty()) {
                 cpgResult.logs.forEach { log ->
                     when {
-                        log.contains("ERROR") || log.startsWith("Exception:") -> p {
+                        log.contains("ERROR") || log.startsWith("Exception:") -> pre {
```

Comment:
Not related with CPG, but ok.
BTW - what is ‘pre’ ? I see it for the first time

Reply:
not formatted: now we print exception with end of lines